# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/65_metamail-2.7-usesox.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/bin/showaudio.usesox	Thu Feb  1 12:31:32 2001
+++ mm2.7/src/bin/showaudio	Thu Feb  1 12:36:24 2001
@@ -151,6 +151,9 @@
 #  Solaris 2.0
     /bin/audioconvert -i raw,voice -f sun $* | /bin/audioplay
     exit $status
+else if (-e /usr/bin/play) then
+# Red Hat Linux
+    exec /usr/bin/play $*
 else
     set dev=/dev/audio
 endif
